openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -691,35 +691,35 @@ If the Access Token is valid for requesting issuance of multiple Credentials, it
 
 ## Binding the Issued Credential to the Identifier of the End-User Possessing that Credential {#credential-binding}
 
-Issued Credential SHOULD be cryptographically bound to the identifier of the End-User who possesses the Credential. Cryptographic binding allows the Verifier to verify during the presentation of a Credential that the End-User presenting a Credential is the same End-User to whom that Credential was issued. For non-cryptographic type of binding and Credentials issued without any binding, see Implementation Considerations in (#claim-based-binding) and (#no-binding).
+An issued Credential SHOULD be cryptographically bound to the identifier of the End-User who possesses the Credential. Cryptographic binding allows the Verifier to verify during the presentation of a Credential that the End-User presenting a Credential is the same End-User to whom that Credential was issued. For non-cryptographic types of binding and Credentials issued without any binding, see the Implementation Considerations in (#claim-based-binding) and (#no-binding).
 
-Note: Claims in the Credential are usually about the End-User who possesses it, but can be about another entity.
+Note: Claims in the Credential are about the subject of the Credential, which is often End-User who possesses it.
 
-For cryptographic binding, the Client has the following options to provide cryptographic binding material for a requested Credential as defined in (#credential_request):
+For cryptographic binding, the Client has the following options to provide cryptographic binding material for a requested Credential, as defined in (#credential_request):
 
-1. Provide proof of control alongside key material (`proof` that includes `sub_jwk` or `did`)
-1. Provide only proof of control without the key material (`proof` that does not include `sub_jwk` or `did`)
+1. Provide proof of control alongside key material (`proof` that includes `sub_jwk` or `did`).
+1. Provide only proof of control without the key material (`proof` that does not include `sub_jwk` or `did`).
```

Comment:
These two options don't make sense to me and don't seem to be defined/explained in the following section (as suggested in the line above). `sub_jwk` doesn't even appear anywhere else in the document AFAIK. 
```suggestion
```

Reply:
I don't feel strongly about it but would still say it's editorial. It's cleaning up non-normative text that seems to have drifted out of sync with the rest of the document (or maybe has always been problematic). And cleaning it up by removing it because I have no idea how to fix it because I don't understand what it is trying to say.

Reply:
It also correlates with the change in the line above 2dbf9326aa96eea6e0a2f836a5e39b10110bafe8

Reply:
FWIW, I agree with the deletion, for the reasons Brian stated.  If people concur that it's non-normative, I'll accept the change into this PR.

Reply:
Im not against removing this text and agree it isn't normative, but I think the paragraph above it that is the pre-text for these options would also need to be remove. If instead this is kept we should fix the typo around the number list which is currently rendering 1. 1. instead of 1. 2.

Reply:
I would like to keep these sentences because they make it clear that there is an option to issue a credential that is not bound to any cryptographically key material.

Reply:
> the paragraph above it that is the pre-text for these options would also need to be remove

That paragraph/sentence had been adjusted with https://github.com/openid/OpenID4VCI/commit/2dbf9326aa96eea6e0a2f836a5e39b10110bafe8 to make sense without the list following. 

> typo around the number list which is currently rendering 1. 1. instead of 1. 2.

AFAIK that's okay markdown syntax and it renders fine - see [that part in draft 12](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0-12.html#section-7.1-3)

> I would like to keep these sentences because they make it clear that there is an option to issue a credential that is not bound to any cryptographically key material.

I did not find that clarity in the text at all.

Reply:
discussed during the Jan-23-2024 WG call. in this PR, agreed to remove the `sub_jwk` reference and potentially open a new issue/PR to discuss removing these two sentences.